.env.example

```diff
@@ -0,0 +1,15 @@
+# GitHub API Token for fetching releases (optional but recommended for rate limits)
+GITHUB_TOKEN=your_github_token_here
+
+# OpenAI API Key for summarizing release notes (optional)
+OPENAI_API_KEY=your_openai_api_key_here
+
+# AWS Configuration for S3 uploads
+# Note: AWS_ROLE_ARN and AWS_REGION are used in GitHub Actions CI only (set as repository secrets)
+# For local development, use AWS CLI profiles or environment variables
+S3_BUCKET=files.ably.io
```

Comment:
Can we upload to `ably-analytics-data-lake-production`, we have existing CSV import pipelines that we can use to ingest this data, but these rely on the data being present in this location.

I will need to provide you with an alternative `AWS_ROLE_ARN` that has permissions to write to this bucket.

Reply:
Sure, happy to do this, or just issue a change on this PR if you want. I have no preference where it should go, just that it needs to be in S3 somewhere, and I expect @zychua25 would make this work anyway when he sets up imports into Snowflake.

Reply:
Ok, I've pushed two commits to remove the sql related code as this will be handled by a data pipeline elsewhere and externalise the S3 config in the action.

Reply:
Thanks @emptyhammond